man/systemd.exec.xml

```diff
@@ -1682,10 +1682,29 @@
       </varlistentry>
 
       <varlistentry>
+        <term><varname>DataDirectory=</varname></term>
+        <term><varname>CacheDirectory=</varname></term>
+        <term><varname>LogDirectory=</varname></term>
```

Comment:
sd-path.c calls this "Logs" rather than just "Log", hence I think we should use the plural s here too